akka-cluster/src/multi-jvm/scala/akka/cluster/StreamRefSpec.scala

```diff
@@ -244,7 +245,14 @@ abstract class StreamRefSpec extends MultiNodeSpec(StreamRefSpec) with MultiNode
         streamLifecycle1.expectMsg("failed-system-42-tmp")
       }
       runOn(third) {
-        streamLifecycle3.expectMsg("failed-system-42-tmp")
+        // there's a race here, we know the SourceRef actor was started but we don't know if it
+        // got the remote actor ref and watched it terminate or if we cut connection before that
+        // and it triggered the subscription timeout. Therefore we must wait more than the
```

Comment:
> or if we cut connection before (..x..) Therefore we must wait

Similar-ish issues to multi-jvm test timings elsewhere , nice it's not needing a direct `Thread.sleep`